lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user) do
```

Comment:
The email will have to render the link and the link will contain the token value, so we'll need the token here as well.